integration-cli/docker_cli_daemon_test.go

```diff
@@ -2538,11 +2538,11 @@ func (s *DockerDaemonSuite) TestRunWithRuntimeFromCommandLine(c *check.C) {
 
 	// Check that we can't override the default runtime
 	s.d.Stop()
-	err = s.d.Start("--add-runtime", "default=docker-runc")
+	err = s.d.Start("--add-runtime", "runc=my-runc")
 	c.Assert(err, check.NotNil)
 
 	content, _ := ioutil.ReadFile(s.d.logFile.Name())
-	c.Assert(string(content), checker.Contains, `runtime name 'default' is reserved`)
+	c.Assert(string(content), checker.Contains, `runtime name 'runc' is reserved`)
```

Comment:
If the user can override it, then their binary will be used instead of ours and the only way to revert would be for them to redefine it with the proper  path.